pom.xml

```diff
@@ -55,6 +55,8 @@
         <okhttp.version>3.8.0</okhttp.version>
         <ant.version>1.10.1</ant.version>
         <javax-el.version>3.0.1-b08</javax-el.version>
+        <h2-mvstore.version>1.4.196</h2-mvstore.version>
+        <ezid.version>1.0.2-SNAPSHOT</ezid.version>
```

Comment:
Can we use version 1.0.1?  It looks like the -SNAPSHOT isn't needed.

Also, if we add the [repo](http://dev-testing.dataone.org/maven) to the RMap parent pom per their [README](https://github.com/NCEAS/ezid), travis builds should work (i.e. be able to resolve the artifact).